config/locales/en/errors.yml

```diff
@@ -613,11 +613,17 @@ en:
               blank: Enter how much is left to pay on the mortgage
               not_a_number: How much there is left to pay on the mortgage must be a number, like 50
             percentage_applicant_owned:
+              invalid: Percentages entered need to total 100%
               blank: Enter the percentage of the property your client owns
-              not_a_number: The percentage of the property your client owns must be a number between 0 and 100
+              not_a_number: &property_client_percentage_error The percentage of the property your client owns must be a number between 1 and 100
```

Comment:
Error message sheet is still 0 - 100. I think user should be able to add as small as 0.01%